### PR TITLE
Some zh_CN translations

### PR DESCRIPTION
**Pull request by VemboLiu**
zh-CN translation for Preference_System_Toc and AddonSIdeBar

### CODE REVIEW

**Comment by peterbe** [comment]
Hi @VemboLiu Can you help me test this? I.e. what am I supposed to look for that wasn't there before and what should I expect to be able to see and I switch to this branch?

**Comment by VemboLiu** [comment]
> Hi @VemboLiu Can you help me test this? I.e. what am I supposed to look for that wasn't there before and what should I expect to be able to see and I switch to this branch?

I'm sorry that I didn't really get you.If you are talking about this request,actually I'm still unfamiliar with travis-ci ,the error is because that I didn't test on my laptop before I pull request and then I found that thre is one more file which I needed to change but I didn't.
So I think maybe I'll pull a new request to deal with the failed check in my spare time? I don't really know.

File: macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -393,6 +393,104 @@ var text = mdn.localStringMap({
...
     'Forums': '부가 기능 포럼',
     '#Contact_us': '연락하기'
+  },
+  'zh-CN': {
+    'WebExtensions': '浏览器扩展程序',
+    'WebExtensions#Getting_started': '开始',
+    'WebExtensions/What_are_WebExtensions': '什么是扩展程序?',
+    'WebExtensions/Your_first_WebExtension': '你的第一个扩展程序',
+    'WebExtensions/Your_second_WebExtension': '你的第二个扩展程序',
+    'WebExtensions/Anatomy_of_a_WebExtension': '详解扩展程序,
```
Comment (peterbe) [inline macros/AddonSidebar.ejs]: This is why the Travis CI is failing. A SyntaxError in your `.ejs`` file.

Comment (VemboLiu) [reply macros/AddonSidebar.ejs]: Thanks,you're right.I'm too careless~I‘ve fixed them and passed now~

**Pull request merged**